private/pkg/app/appcmd/appcmdtesting/appcmdtesting.go

```diff
@@ -115,8 +115,18 @@ func RunCommandExitCode(
 	// make the use something different than the actual command
 	// to make sure that all code is binary-name-agnostic.
 	use := "test"
-	stderrCopy := bytes.NewBuffer(nil)
 	stdoutCopy := bytes.NewBuffer(nil)
+	if stdout == nil {
```

Comment:
This came up as a panic in the tests when I had `stdout` as nil - this makes it so that `stdout/stderr` can be nil